integration_tests/test_str_attributes.py

```diff
@@ -366,7 +366,69 @@ def is_space():
     s = ""
     assert s.isspace() == False
 
+def is_alnum():
```

Comment:
For future, I would suggest adding changes in integration tests in a separate commit. That can/would help avoid/resolve conflicts in integration tests. If there are conflicts and you are unable to resolve them, you can simply drop the specific conflicting commit and then rebase without any conflicts. Later manually add changes for the dropped commit.

I am sharing this since you experienced some conflicts in your pervious PR https://github.com/lcompilers/lpython/pull/2572.

Reply:
Ok, I'll follow this from now on. Thank you.